.travis.yml

```diff
@@ -5,6 +5,10 @@ services:
 
```

Comment:
just ran it through my localhost and I LOVE it ! It looks really great. I did have to run the tables in different order as well, what you commented on mine too, but aside from that everything seems to work perfectly!

Reply:
Thanks :) Creating table order is definitely something I'll make sure to check for next time!